code-reviews/example1/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>All About Broccoli</title>
+    <link rel='stylesheet' type='text/css' href='styles.css'>
+</head>
+<body class='body'>
```

Comment:
wouldn't it be better if we directly select the HTML element? I don't see the usability of the 'body' class